lib/taxonomy.rb

```diff
@@ -42,6 +42,24 @@ def descendants
       tree.tap(&:shift)
     end
 
+    # Get ancestors of a taxon
+    #
+    # @return [Array] all taxons in the path from the root of the taxonomy to the parent taxon
+    def ancestors
```

Comment:
This functionality will be soon removed from here and we will use the gem.
That will be part of code refactor. 

https://github.com/alphagov/govuk_taxonomy_helpers/blob/master/lib/govuk_taxonomy_helpers/linked_content_item.rb#L56-L69